.editorconfig

```diff
@@ -5,8 +5,17 @@ root = true
 [*]
 end_of_line = lf
 insert_final_newline = true
+trim_trailing_whitespace = true
 charset = utf-8
 
-[*.md]
+[*.{md,css,html}]
 indent_style = space
 indent_size = 4
+
+[*.toml]
+indent_style = space
+indent_size = 2
+
+[*.go]
+indent_style = tab
+indent_size = 2
```

Comment:
Is the indent_size needed here? We don't have it defined in the server's .editorconfig, so I don't know if it does anything here.

Reply:
BTH it might not be needed there. But it does no harm here.

Reply:
That's fine then. I'll just add Joram as a second reviewer then so we can make sure to get two reviews before merging this